docs/guide.rst

```diff
@@ -1921,15 +1921,14 @@ Dependabot
 
 Dependabot_ creates pull requests with automated dependency updates.
 
-Follow these steps to set up Dependabot for your repository:
+Enable `Security Updates` from the `Security` tab on the repository under `Dependabot alerts` > `Dependabot Security Updates`.
 
-1. Sign up at Dependabot_.
-2. Install their GitHub app.
+More on `official documentation`_ for GitHub Dependabot.
 
 The configuration is included in the repository,
-in the file `.dependabot/config.yml`__.
+in the file `.github/dependabot.yml`__.
 
-__ https://dependabot.com/docs/config-file/
+__ https://help.github.com/en/github/administering-a-repository/customizing-dependency-updates
 
 Dependabot integrates with your repository via its GitHub app.
```

Comment:
While this is still true, it seems better to remove this sentence: GitHub now takes care of enabling and authorizing this app, so this is completely transparent to the user.

```suggestion
```